src/edu/cornell/mannlib/vitro/webapp/controller/individual/IndividualResponseBuilder.java

```diff
@@ -86,7 +86,15 @@ ResponseValues assembleResponse() throws TemplateModelException {
 		Individual dcoId = individual.getRelatedIndividual("http://info.deepcarbon.net/schema#hasDcoId");
 		if (dcoId != null) {	
 			body.put("dcoId", dcoId.getRdfsLabel());
+			System.out.println("world!!!");
 		}
+		// Trial for networkId
+		Individual networkId = individual.getRelatedIndividual("http://info.deepcarbon.net/schema#networkId");
```

Comment:
yes i just realized and changed it. and exactly here i have a question, it seems we should be using 
getObjectPropertyStatements() other than getRelatedIndividual()